### DISCUSSION TITLE
Snowflake Integration

### DISCUSSION THREAD

**Discussion by fxmb** [feast-dev/feast#2052] [Q&A]
Hello team 👋,

I was wondering when the for Q4 2021 announced Snowflake integration will be available? Is this still on the roadmap?
Also, are there any examples for working Snowflake setups somewhere?

Thanks for the help and information.

**Comment by woop** [comment]
Hey @franzbewerunge. The connector is actively being developed by folks over at Snowflake. I believe they will be releasing a first version pretty soon (in the next month or two). We don't have an exact date yet.

**Comment by sfc-gh-madkins** [comment]
Hey @franzbewerunge -- I currently have a fork with all the snowflake work here: sfc-gh-madkins/feast

You should be able to have snowflake as both and offline store, online store, or both via the snowflake provider.

I recommending start with: `feast init -t snowflake` and take the template for a spin. Remember case sensitivities when typing in snowflake credentials.

**Reply by fxmb** [reply]
Thanks for the reply and the help! Is this "just" a community solution or will this be part of the officially planned snowflake integration?

**Comment by sfc-gh-madkins** [comment]
This is part of the official snowflake integration.
